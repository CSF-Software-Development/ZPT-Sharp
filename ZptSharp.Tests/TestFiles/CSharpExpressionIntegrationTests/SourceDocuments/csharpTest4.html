<html>
<head>
<title>CSharp aliased namespace test</title>
</head>
<body>
<p tal:define="throwaway using:ci System.Globalization.CultureInfo">
  The current thread culture is:
  <span tal:content="csharp:ci.GetCultureInfo(&quot;en-GB&quot;).ToString()">fr-FR</span>
</p>
</body>
</html>