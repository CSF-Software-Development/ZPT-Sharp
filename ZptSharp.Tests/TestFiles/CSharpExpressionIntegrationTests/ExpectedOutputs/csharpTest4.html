<html>
<head>
<title>CSharp aliased namespace test</title>
</head>
<body>
<p>
  The current thread culture is:
  <span>en-GB</span>
</p>
</body>
</html>